### PR TITLE
Add a test that uses an inherited struct in another nested struct.

### PR DESCRIPTION
**Pull request by baldersheim**
@lesters PR

### CODE REVIEW

File: tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (lesters) [inline tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java]: 2020

File: tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import com.yahoo.document.*;
+import com.yahoo.document.annotation.SpanTree;
+import com.yahoo.docproc.*;
+import com.yahoo.concretedocs.Base;
+import com.yahoo.concretedocs.Usebase;
+import java.util.logging.Logger;
+
+/**
+ * An dummy document processor for testing concrete struct inheritance.
```
Comment (lesters) [inline tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java]: A

File: tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java (hunk 1 above)
Comment (baldersheim) [reply tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java]: Fixed

File: tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java (hunk 2 above)
Comment (baldersheim) [reply tests/search/struct_and_map_types/inherited_struct/concretedocs/src/main/java/concretedocs/ConcreteDocDocProc.java]: Fixed

**Pull request merged**